### PR TITLE
Android SDK 5.7.1.1267

### PR DESCRIPTION
**Pull request by zvs001**
I was looking for a while to find how to get rid from .arr files.
I solved finally it with jitpack and upgraded SDKs.

SDK upgrade to 5.7.1 can be critical, because Zoom sent emails that google play can remove apps from google play  with lower sdk version. (policy issues).

It have few breaking updates, so I think it is better to start v6. I don't see any real case to keep version same like zoom. Because developers always have to upgrade to latest.

### CODE REVIEW

**Review by mieszko4 — APPROVED** [review]
:rocket: Thnx! I will merge and test this it a bit later.

I've tried a few times to upgrade android sdk but I got stucked with some architecture native issues.
Great idea to move out .aar out of the lib!

Could you confirm that the smoke test works? I.e. joining the meeting? :)

**Comment by zvs001** [comment]
Yes. I tested it in debug and both worked fine.
I will wait for merge and I will test release version too.

I think it should be ok

**Comment by sumeetHalder** [comment]
hi @mieszko4 ,

Is there any plan to update IOS SDK to 5.7.1.., Will be help full if you could do this as well.
thanks!!

**Comment by mieszko4** [comment]
Yes, I've created a new issue: https://github.com/mieszko4/react-native-zoom-us/issues/109

**Comment by zvs001** [comment]
I will try to add IOS sdk soon

File: android/src/main/java/ch/milosz/reactnative/RNZoomUsModule.java [hunk 1]
```diff
@@ -247,8 +251,15 @@ public void run() {
...
           MeetingViewsOptions view = new MeetingViewsOptions();
-          if(paramMap.hasKey("participantID")) opts.participant_id = paramMap.getString("participantID");
           if(paramMap.hasKey("noAudio")) opts.no_audio = paramMap.getBoolean("noAudio");
+          /**
+              participant_id was removed from android options.
+              There is no propper documentations and it still exists in jave docs...
+              Maybe it was renamed to customer_key or so on. (todo check)
+              Waiting before further changes.
+           */
+//          if(paramMap.hasKey("participantID")) opts.participant_id = paramMap.getString("participantID");
```
Comment (mieszko4) [inline android/src/main/java/ch/milosz/reactnative/RNZoomUsModule.java:261]: based on https://devforum.zoom.us/t/participant-id-generated-by-me-and-passed-in-zoommtg-join-is-changing-to-zoom-generated-id-websdk/18018/4 `participant_id` has been deprecated and will be removed in web sdk.
So I guess it applies to all sdks. The alternative is to use `email`

Comment (mieszko4) [reply android/src/main/java/ch/milosz/reactnative/RNZoomUsModule.java:261]: it looks like it has been renamed to `customer_key` (https://marketplacefront.zoom.us/sdk/meeting/android/us/zoom/sdk/MeetingOptions.html#customer_key)

**Pull request merged**